ext/standard/dns.c

```diff
@@ -400,8 +401,9 @@ PHP_FUNCTION(dns_check_record)
 		else if (!strcasecmp("NAPTR", rectype)) type = DNS_T_NAPTR;
 		else if (!strcasecmp("A6",    rectype)) type = DNS_T_A6;
 		else {
-			php_error_docref(NULL, E_WARNING, "Type '%s' not supported", rectype);
-			RETURN_FALSE;
+			zend_argument_value_error(2, "must be one of 'A', 'NS', 'MX', 'PTR', 'ANY', 'SAO', 'CAA', "
```

Comment:
Please use `"` instead of `'` so that we'll have a consistent format once https://github.com/php/php-src/pull/5590 is merged :)

Reply:
Hmm... Wouldn't it be possible to use the constants below here as well?

Reply:
The constants are integers, whereas this function accepts a string, so no

Reply:
Ah, I see! Thanks